src/couch_jobs/src/couch_jobs_server.erl

```diff
@@ -95,13 +96,22 @@ handle_info(check_types, St) ->
     {noreply, St};
 
 handle_info({'DOWN', _Ref, process, Pid, Reason}, St) ->
+    ?LOG_ERROR(#{
+        what => unknown_process_crash,
```

Comment:
The `unknown_process_crash` is ambiguous. It could be understood as:

- the process we don't know about crashed
- the process crashed due to unknown reason

In this case it is either `notifier` or `monitor` process. Maybe we should say something like:

```
what => monitored_process_crash,
```